src/components/Admin/index.jsx

```diff
@@ -309,6 +316,13 @@ class Admin extends React.Component {
                   <h2>Overview</h2>
                 </div>
               </div>
+              <div className="row mt-4">
+                <div className="col">
+                  {insightsLoading ? <AIAnalyticsSummarySkeleton /> : (
+                    insights && <AIAnalyticsSummary enterpriseId={enterpriseId} />
```

Comment:
We should an error message if `insights` are not loaded for some reason.

Reply:
@saleem-latif as discussed with @muhammad-ammar, we are not showing `insights` section if we have not received data or encountered some error. Please let me know if this needs to change.

Reply:
@mahamakifdar19 Here is what I think the current behavior is when there is API error, let me know if my understanding is not correct.

User clicks on the button to view analytics, we will show the user loading card and when the call fails we will remove the loading card from the DOM without any indication of what happened.

I think that will confuse the users, if we instead show an error message after removing the loading card that the API call failed and user can try again later, then it would be a better UX.

Reply:
@saleem-latif this section is for displaying analytics buttons "Summarize Analytics" and "Track Progress" so if there is an error or no data available, we will restrict showing the entire section.